src/components/App/App.jsx

```diff
@@ -63,58 +76,82 @@ function App() {
 
   return (
     <AppGrid>
-      <Countdown minutes={minutes} seconds={seconds} />
-      <LengthButtonGrid>
-        <TimerLength
+      <CountdownGrid>
+        <Countdown minutes={minutes} seconds={seconds} />
+        {isActive ? (
```

Comment:
Is there a reason for returning an empty string here? Not sure, but I think you could simply replace the ternary by an `&&` operator. 
`{isActive && <EndTime endHours={endHours} endMinutes={endMinutes} />}`

Reply:
Thank you for the recommendation.
On javascript.info, I have read to not replace `&&`with `if`, due to readability and understanding purposes. 

Take a look to the end of the section `&&` under the following link and let's discuss afterwards, if you understand it the same way ☺️: [JS info - &&](https://javascript.info/logical-operators#and-finds-the-first-falsy-value)